config/steampipecloud.spc

```diff
@@ -0,0 +1,6 @@
+connection "steampipecloud" {
+  plugin   = "steampipecloud"
+
+# Access Token for your SteampipeCloud account
+  token = "YOUR_STEAMPIPECLOUD_ACCESS_TOKEN"
+}
```

Comment:
```suggestion
connection "steampipecloud" {
  plugin = "steampipecloud"

  # Steampipe Cloud API token. If `token` is not specified, it will be loaded
  # from the `STEAMPIPE_CLOUD_TOKEN` environment variable.
  # token = "YOUR_STEAMPIPECLOUD_ACCESS_TOKEN"
}
```